src/select/select.tsx

```diff
@@ -105,7 +107,7 @@ class Select<T> extends React.Component<Props<T>, State<T>> {
         );
     }
 
-    private getFilter = () => {
+    private getFilter() {
```

Comment:
@efekaptan I just aligned it with the existing methods. If you think it's worth changing the current conventions: let's talk about it and evaluate.

Reply:
As this function is not triggered by an event, `this` scope is not an issue. It can be aligned, ok.